custom/conf/app.example.ini

```diff
@@ -1518,32 +1538,17 @@ ROUTER = console
 ;; Prefix displayed before subject in mail
 ;SUBJECT_PREFIX =
 ;;
-;; Mail server protocol. One of "smtp", "smtps", "smtp+startls", "smtp+unix", "sendmail", "dummy".
-;; - sendmail: use the operating system's `sendmail` command instead of SMTP. This is common on Linux systems.
-;; - dummy: send email messages to the log as a testing phase.
-;; If your provider does not explicitly say which protocol it uses but does provide a port,
-;; you can set SMTP_PORT instead and this will be inferred.
-;; (Before 1.18, see the notice, this was controlled via MAILER_TYPE and IS_TLS_ENABLED.)
-;PROTOCOL =
-;;
-;; Mail server address, e.g. smtp.gmail.com.
-;; For smtp+unix, this should be a path to a unix socket instead.
-;; (Before 1.18, see the notice, this was combined with SMTP_PORT as HOST.)
-;SMTP_ADDR =
-;;
-;; Mail server port. Common ports are:
-;;   25:  insecure SMTP
-;;   465: SMTP Secure
-;;   587: StartTLS
-;; If no protocol is specified, it will be inferred by this setting.
-;; (Before 1.18, this was combined with SMTP_ADDR as HOST.)
-;SMTP_PORT =
-;;
-;; Enable HELO operation. Defaults to true.
-;ENABLE_HELO = true
-;;
-;; Custom hostname for HELO operation.
-;; If no value is provided, one is retrieved from system.
+;; Mail server
+;; Gmail: smtp.gmail.com:587
+;; QQ: smtp.qq.com:465
+;; As per RFC 8314 using Implicit TLS/SMTPS on port 465 (if supported) is recommended,
+;; otherwise STARTTLS on port 587 should be used.
+;HOST =
+;;
+;; Disable HELO operation when hostnames are different.
+;DISABLE_HELO =
+;;
+;; Custom hostname for HELO operation, if no value is provided, one is retrieved from system.
```

Comment:
```suggestion
;; Mail server protocol. One of "smtp", "smtps", "smtp+startls", "smtp+unix", "sendmail", "dummy".
;; - sendmail: use the operating system's `sendmail` command instead of SMTP. This is common on Linux systems.
;; - dummy: send email messages to the log as a testing phase.
;; If your provider does not explicitly say which protocol it uses but does provide a port,
;; you can set SMTP_PORT instead and this will be inferred.
;; (Before 1.18, see the notice, this was controlled via MAILER_TYPE and IS_TLS_ENABLED.)
;PROTOCOL =
;;
;; Mail server address, e.g. smtp.gmail.com.
;; For smtp+unix, this should be a path to a unix socket instead.
;; (Before 1.18, see the notice, this was combined with SMTP_PORT as HOST.)
;SMTP_ADDR =
;;
;; Mail server port. Common ports are:
;;   25:  insecure SMTP
;;   465: SMTP Secure
;;   587: StartTLS
;; If no protocol is specified, it will be inferred by this setting.
;; (Before 1.18, this was combined with SMTP_ADDR as HOST.)
;SMTP_PORT =
;;
;; Enable HELO operation. Defaults to true.
;ENABLE_HELO = true
;;
;; Custom hostname for HELO operation.
;; If no value is provided, one is retrieved from system.
```